CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ### Unreleased
 
+* Extend MetricsAgent to handle additional metric types
+* Add StatsiteAgent capable of formatting and sending UDP messages to [statsite](https://github.com/statsite/statsite)
+
```

Comment:
Maybe mention DateTimeDiff (and why) too?

Reply:
Ah yes, forgot about that. Try articulating the problem that solves!

Reply:
> Add `DateTimeDiff::microsBetween()` to calculate the exact number of microseconds between two DateTimeImmutable values without any risk of floating point precision errors.

Or something?